x-pack/test/api_integration/apis/metrics_ui/services.ts

```diff
@@ -7,24 +7,41 @@
 
 import expect from '@kbn/expect';
 import { ServicesAPIResponseRT } from '@kbn/infra-plugin/common/http_api/host_details';
+import { ApmSynthtraceEsClient, createLogger, LogLevel } from '@kbn/apm-synthtrace';
 import { decodeOrThrow } from '@kbn/infra-plugin/common/runtime_types';
-import { FtrProviderContext } from './types';
+import { FtrProviderContext } from '../../ftr_provider_context';
 import { generateServicesData, generateServicesLogsOnlyData } from './helpers';
 
 const SERVICES_ENDPOINT = '/api/infra/services';
 
 export default function ({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
-  const synthtrace = getService('apmSynthtraceEsClient');
+  const apmSynthtraceKibanaClient = getService('apmSynthtraceKibanaClient');
+  const esClient = getService('es');
+
   describe('GET /infra/services', () => {
+    let synthtraceApmClient: ApmSynthtraceEsClient;
     const from = new Date(Date.now() - 1000 * 60 * 2).toISOString();
     const to = new Date().toISOString();
+    before(
+      async () =>
+        (synthtraceApmClient = new ApmSynthtraceEsClient({
```

Comment:
Could this instantiation be encapsulated in a utils function?

Reply:
I meant the latter. Something like [this](https://github.com/elastic/kibana/blob/main/x-pack/test/apm_api_integration/common/bootstrap_apm_synthtrace.ts#L17). The way the client is instantiated probably won't change across the tests that will use it, so a function like that could make it easier to use the client - it doesn't make a too much difference, it's just an idea.

Reply:
Thanks, Carlos. Let me know if this is better.  Can make it more flexible at the point when someone needs it.